src/App.js

```diff
@@ -34,15 +34,15 @@ function App() {
   }
 
   const addAppointment = ({title, contact, date, time}) => {
```

Comment:
I was wondering if they both work and if so, which is the preferred method

Reply:
It depends 😅.

If there are a tonne of arguments, it can be nice bundling them together in an object, but when you pass that object to the function, it can be confusing what exactly you're passing. 

I think in this case a single object is fine because all the attributes are appointment details and relate closely to the one appointment.